apps/wallet/src/ui/app/components/nft-display/index.tsx

```diff
@@ -53,46 +54,65 @@ function NFTDisplayCard({
         </div>
     );
 
-    const defaultSection = (
-        <>
-            {expandable ? (
-                <div className={st.expandable}>
-                    <ExplorerLink
-                        type={ExplorerLinkType.object}
-                        objectID={nftObjectID}
-                        showIcon={false}
-                        className={st['explorer-link']}
-                    >
-                        View Image <Icon icon={SuiIcons.Preview} />
-                    </ExplorerLink>
-                </div>
-            ) : null}
-            {showlabel && displayTitle ? (
-                <div className={st.nftfields}>{displayTitle}</div>
-            ) : null}
-        </>
-    );
+    const defaultSection =
+        showlabel && displayTitle ? (
+            <div
+                className={cl(
+                    'items-center mt-2 text-steel-dark',
+                    animateHover &&
+                        'group-hover:text-black duration-200 ease-ease-in-out-cubic'
+                )}
+            >
+                {displayTitle}
+            </div>
+        ) : null;
 
+    const borderRadiusCl = borderRadius === 'md' ? 'rounded' : 'rounded-[10px]';
+    const borderRadiusHoverCl =
+        borderRadius === 'md' ? 'rounded-sm' : 'rounded-[5px]';
+    const mediaContainerCls = animateHover
+        ? `ease-ease-out-cubic duration-[400ms] group-hover:shadow-steel/50 group-hover:shadow-[0_0_20px_0] ${borderRadiusCl} hover:${borderRadiusHoverCl}`
+        : '';
+    const mediaCls = animateHover
+        ? 'group-hover:scale-[115%] duration-500 ease-ease-out-cubic'
+        : '';
     return (
-        <div className={cl(st.nftimage, wideview && st.wideview, st[size])}>
-            {filePath ? (
-                <img
-                    className={cl(st.img)}
-                    src={filePath}
-                    alt={fileExtensionType?.name || 'NFT'}
-                    title={nftTypeShort}
-                />
-            ) : (
-                <div className={st.noMedia} title={nftTypeShort}>
-                    <Icon
-                        className={st.noMediaIcon}
-                        icon={SuiIcons.NftTypeImage}
-                    />
-                    {wideview ? null : (
-                        <span className={st.noMediaText}>No media</span>
-                    )}
-                </div>
+        <div
+            className={cl(
+                st.nftimage,
+                wideview && st.wideview,
+                st[size],
+                'group'
             )}
+        >
+            <div
+                className={cl(
+                    'flex flex-shrink-0 items-stretch flex-grow self-stretch overflow-hidden',
+                    mediaContainerCls
+                )}
+            >
+                {filePath ? (
+                    <img
+                        className={cl(st.img, 'rounded-none', mediaCls)}
```

Comment:
IMO you should either keep using CSS modules or convert everything to tailwind, the in-between is rough. This applies everywhere.